src/build.props

```diff
@@ -35,9 +35,11 @@
   <PropertyGroup Label="Package">
     <PackageId Condition=" '$(PackageId)' == '' ">$(AssemblyName)</PackageId>
     <Authors Condition=" '$(Authors)' == '' ">$(Company)</Authors>
+    <Owners Condition=" '$(Authors)' == '' ">$(Company),tse-securitytools</Owners>
     <PackageRequireLicenseAcceptance Condition=" '$(PackageRequireLicenseAcceptance)' == '' ">false</PackageRequireLicenseAcceptance>
     <PackageLicenseUrl>https://github.com/Microsoft/jschema/blob/master/License.txt</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/Microsoft/jschema</PackageProjectUrl>
+    <PackageIconUrl>https://go.microsoft.com/fwlink/?linkid=2009431</PackageIconUrl>
```

Comment:
>microsoft [](start = 31, length = 9)

Any reason not to use the raw.github link?

Reply:
hm was advised to use a forwarder, but probably not necessary to do so.

---
In reply to: [209795228](https://github.com/Microsoft/jschema/pull/21#discussion_r209795228) [](ancestors = 209795228)

Reply:
No, that's ok, I didn't know there was some standard practice in play here.

---
In reply to: [209798849](https://github.com/Microsoft/jschema/pull/21#discussion_r209798849) [](ancestors = 209798849,209795228)